connectors/services/job_scheduling.py

```diff
@@ -167,29 +170,40 @@ async def _run(self):
                 await self.sync_job_index.close()
         return 0
 
-    async def _scheduled_sync(self, connector, job_type):
+    async def _try_schedule_sync(self, connector, job_type):
+        last_wake_up_time = self.last_wake_up_time
```

Comment:
Question: Do we need this assignment? Is the rationale to shorten the name throughout the method by removing the `self` prefix?

Reply:
It's just personal taste - since it's used several times, I just put it to variable and use down the code. So purely personal style, not opposed to inlining it.